### PR TITLE
Fix some bugs of The Jeffrey Consolation

### LINKED ISSUE #135: No Jeff in The Jeffrey Consolation
Jeff won't appear in The Jeffrey Consolation if he was killed in the most recent playthrough of The Source.

### PR DESCRIPTION
**Pull request by moonysolari**
- Fixes #135 .
- Also fixes the bug where the objectives are not visible on the planning page for levels 2 and 3.
- Does not fix the bug that there is an empty objective in the in-game F1 objective list for levels 2 and 3.
- Overall, this PR makes this escalation more playable, but it is still not perfect and still has details that need fixing.

### CODE REVIEW

File: contractdata/SAPIENZA/JEFFREY1.json [hunk 1]
```diff
@@ -78,7 +78,9 @@
                 }
             }
         ],
-        "Bricks": [],
+        "Bricks": [
+            "assembly:/_PRO/Scenes/Missions/CoastalTown/override_constantjeff.brick"
```
Comment (RDIL) [inline contractdata/SAPIENZA/JEFFREY1.json:82]: Where is this from?

Comment (AnthonyFuller) [reply contractdata/SAPIENZA/JEFFREY1.json:82]: Official, was introduced this(?) patch.

Comment (RDIL) [reply contractdata/SAPIENZA/JEFFREY1.json:82]: For contracts or what?

Comment (moonysolari) [reply contractdata/SAPIENZA/JEFFREY1.json:82]: Yeah, it's for contracts.

**Review by RDIL — APPROVED** [review]

**Pull request merged**